packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.test.js

```diff
@@ -49,4 +49,45 @@ describe('SimpleTextKeyValue', () => {
 		const wrapper = shallow(<SimpleTextKeyValue value="myValue" />);
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
+	it('should render the type', () => {
+		const schema = {
+			type: {
+				type: 'int',
+			},
+		};
+		const data = {
+			value: '',
+		};
+		const wrapper = shallow(
+			<SimpleTextKeyValue
+				className="myCLass"
+				formattedKey="myKey"
+				value={data}
+				schema={schema}
+				displayTypes
+			/>,
+		);
+		expect(wrapper.find('.tc-simple-text-type').text()).toBe('- int');
+	});
+	it('should render the type with a custom render', () => {
+		const schema = {
+			type: {
+				type: 'int',
+			},
+		};
+		const data = {
+			value: '',
+		};
+		const wrapper = shallow(
+			<SimpleTextKeyValue
+				className="myCLass"
+				formattedKey="myKey"
+				value={data}
+				schema={schema}
+				displayTypes
+				typesRenderer={s => <span>And the type is {s.type.type}</span>}
+			/>,
+		);
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
```suggestion
		expect(wrapper.find('.tc-simple-text-type').text()).toBe('And the type is int');
```

Can we avoid the snapshot please ?

If you really want to check that the span is hear, perhaps you can snapshot just `wrapper.find('.tc-simple-text-type').getElement()`